commerce-services-sample/src/main/java/com/godaddy/commerce/services/sample/catalog/category/create/CategoryCreateViewModel.kt

```diff
@@ -17,80 +26,134 @@ import com.godaddy.commercecore.models.CategoryProduct
 import com.godaddy.commercecore.models.CategoryTreeNode
 import kotlinx.coroutines.FlowPreview
 import kotlinx.coroutines.suspendCancellableCoroutine
+import timber.log.Timber
 
 class CategoryCreateViewModel : CommonViewModel<CategoryCreateViewModel.State>(State()) {
 
     private val catalogServiceClient = getCatalogService(viewModelScope)
 
+    init {
+        loadProducts()
+    }
+
+    private fun loadProducts(query: String? = null) {
+        execute {
+            val service = catalogServiceClient.getService().getOrThrow()
+            val bundle = Bundle().apply {
+                putParcelable(ProductParams.DATA_SOURCE, DataSource.REMOTE_IF_EMPTY)
+                putInt(ProductParams.PAGE_OFFSET, 0)
+                putInt(ProductParams.PAGE_SIZE, 100)
+                putString(ProductParams.SORT_BY, CatalogContract.Product.Columns.UPDATED_AT)
+                putString(ProductParams.SEARCH_TERM, query)
+            }
```

Comment:
Change to use ProductParamsExt kotlin extension. It enforces strong typing.

```suggestion
            val bundle = ProductParamsExt.toBundle(
                dataSource = DataSource.REMOTE_IF_EMPTY,
                pageOffset = 0,
                pageSize = 100,
                sortBy = CatalogContract.Product.Columns.UPDATED_AT,
                searchTerm = query.nullIfEmpty()
            )
```